src/analytics/segment.js

```diff
@@ -41,10 +41,6 @@ const extraProperties = store => {
   };
 };
 
-const context = {
-  ip: "0.0.0.0",
-};
```

Comment:
please rollback the change on JS side. it's actually a bug in the current segment lib but we must keep the present code.